quic/s2n-quic-rustls/src/lib.rs

```diff
@@ -13,13 +16,14 @@ pub mod certificate;
 pub mod client;
 pub mod server;
 
-pub use cipher_suite::DEFAULT_CIPHERSUITES;
+pub use cipher_suite::default_crypto_provider;
 pub use client::Client;
 pub use server::Server;
 
 //= https://www.rfc-editor.org/rfc/rfc9001#section-4.2
 //# Clients MUST NOT offer TLS versions older than 1.3.
-static PROTOCOL_VERSIONS: &[&rustls::SupportedProtocolVersion] = &[&rustls::version::TLS13];
+pub static TLS13_PROTOCOL_VERSION: &[&rustls::SupportedProtocolVersion] =
```

Comment:
This should not be specific to TLS 1.3, since QUIC will be compatible with >=TLS 1.3

```suggestion
pub static PROTOCOL_VERSIONS: &[&rustls::SupportedProtocolVersion] =
```

Reply:
And do we really need to make this `pub`?

Reply:
Lemme investigate if its possible to hide this.

Reply:
[new change](https://github.com/aws/s2n-quic/pull/2143/commits/c0a81fa6ee9b63d5e9f30c771ad03cde0b61d6d1#diff-2aff5ae4b117c8a1c6051ee3189f3a8a5cdebae9d708095b9b5fa02bdd0b25ffR27) I like the ease of use but it is 2 new APIs so wanted some feedback on what folks think.

One thing I am not sure about is if this should be part of the s2n-quic-rustls::Client struct since its returning crate::rustls::ConfigBuilder. Possibly we rename the fn `default_rustls_config_builder`?